flattentool/input.py

```diff
@@ -555,7 +556,10 @@ def get_sheet_lines(self, sheet_name):
 
 class XLSXInput(SpreadsheetInput):
     def read_sheets(self):
-        self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        try:
+            self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        except zipfile.BadZipFile as e:
+            raise Exception("The supplied file has extension .xlsx but isn't a XSLX file.")
```

Comment:
This should read "isn't *an* XLSX file" (change 'a' to 'an' and correct order of XLSX)